packages/react/src/components/Toolbar/Toolbar-story.js

```diff
@@ -29,7 +29,7 @@ const inputProps = {
   onChange: action('onChange'),
 };
 
-storiesOf('[Deprecated] Toolbar', module).add(
+storiesOf('Toolbar [Deprecated]', module).add(
```

Comment:
If we could leave `[Deprecated]` first that'd be great 👍

Reply:
The issue here is that Storybook loads all of the stories sequentially, resulting in an alphabetic ordering based on file name. This puts the `Grid	` component at the end of the list when it's loading stories in the new and old formats and thus 123123 forcefully sorts it. This incorrectly puts toolbar with the other "D" components. https://github.com/carbon-design-system/carbon/pull/4894/files/acd0d48408f4ad561cf705e69f04b631543118dc#diff-2e6866a078d3aa8cca27fed58050c6cdR35-R36
Referenece: 
https://storybook.js.org/docs/configurations/options-parameter/#sorting-stories